Libraries/Components/DrawerAndroid/DrawerLayoutAndroid.macos.js

```diff
@@ -1,15 +1,14 @@
 /**
- * Copyright (c) 2015-present, Facebook, Inc.
+ * Copyright (c) Facebook, Inc. and its affiliates.
  *
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  *
- * @flow
```

Comment:
Why not just let flow check this stub file too?

Reply:
Because it would fail.   Facebook's DrawerLayoutAndroid.ios.js is the same: it has no `@flow` tag and if added it fails.

Reply:
The facebook .ios.js file also has no @flow.   If added it fails to validate.   We're just matching ios.